src/schema/rules/datatypes/derivatives.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "derivatives_common"
+  # all datatypes
+  additional_entities:
+    space: OPTIONAL
+    desc: OPTIONAL
```

Comment:
A thought for being explicit rather than matching:

```YAML
- $ref:parametric_anat
  additional_entities:
    space: OPTIONAL
    desc: OPTIONAL

- $ref:nonparametric_anat
  additional_entities:
    space: OPTIONAL
    desc: OPTIONAL
```